pnpm-lock.yaml

```diff
@@ -38719,6 +38759,33 @@ snapshots:
     transitivePeerDependencies:
       - ts-node
 
+  tailwindcss@3.4.4(ts-node@10.9.2(@types/node@20.14.6)(typescript@5.5.2)):
```

Comment:
suspicious, do we have different versions of tailwind installed now - if so lets align them